.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,544 @@
+<component name="ProjectCodeStyleConfiguration">

```

Comment:
this should not be commit ? Maybe `.gitignore` should be edited ?

Reply:
No, it's on my side. Some files are automatically included in the git commit. Bothering.... I've to find a way to fix this.